### DISCUSSION TITLE
How many users of Etherpad are out there, do you know?

### DISCUSSION THREAD

**Discussion by weettoni** [ether/etherpad-lite#4860] [Q&A] [answered]
I am doing some research on open source solutions and Etherpad is my lead example for the note pad and collaborative document building. 

Thank you!

**Comment by JohnMcLear** [comment]
It's impossible for us to know how many users there are, it would be an invasion of privacy to know this.  Sorry!

**Comment by webzwo0i** [comment, marked as answer]
If you want to approximate:
https://www.shodan.io/search?query=Etherpad is returning around 3700 instances, around 3500 on port 9001 which makes sense, because it's our default port. Note however, that a lot of people will host etherpad behind a reverse proxy setup/multiple domains on one IP address or in their internal network, and Shodan won't find them (they scan the whole internet on all ports - but you get a number here).
You could then google for "or create/open a Pad with the name". Instances that don't customize their landing page and use English and don't forbid google, currently around 300 can be found this way. For every found domain, you can query the /stats resource. If it's available, use the "mean" value of the "connects" key, which tells you how many users per second connect to the instance.

On https://static.etherpad.org/index.html you can see that around 87k plugins where downloaded last month. ep_align, the most often used plugin, was at 7k downloads. You can't trust those numbers either, because not every instance was restarted last month or the devops may install plugins manually or the instance won't use plugins at all.

**Reply by JohnMcLear** [reply]
Also remember all the onion etc deployments and schools etc who run Etherpad for Lan use only

**Reply by weettoni** [reply]
Thank you @webzwo0i , @JohnMcLear . I will make this work. It is pity this kind of data is not evident, as anyone analyzing the traction and the popularity of the OS tools will have a hard time getting it in. I understand the obstacles: privacy and self-hosting and respect them.

**Comment by Gared** [comment]
I know this question is old, but if someone is also interested in those numbers I started to give some insights.
On https://static.etherpad.org/index.html you will find "Etherpad statistics" about running instances.
Another project that I'm working on is an etherpad scanner that is crawling etherpad instances and getting some insights. You can find the information on this page: https://scanner.etherpad.org/
